docs/runjobs/lumi_env/softwarestacks.md

```diff
@@ -104,13 +104,13 @@ stacks also with `LTS` next to their name which would then denote that this is
 a release that we will try to support long-term (ideally two years), but
 currently the system is changing too rapidly (as some of the hardware is new
 and not an evolution of previous hardware) so we cannot guarantee any level of
-longevity for any of the software stacks. In fact, past experience has shown
+longevity for any of the software stacks. In fact, experience has shown
```

Comment:
The old wording was better

Reply:
I feel "past" is superfluous there, since experience can't be present or future? 😁